rest/data/host_create.go

```diff
@@ -157,6 +162,44 @@ func CreateHostsFromTask(ctx context.Context, env evergreen.Environment, t *task
 	return catcher.Resolve()
 }
 
+// GetDistro returns the distro based on the name and provider.
```

Comment:
Can we mention that for EC2 `host.create`, the distro is optional and this is allowed to return an empty distro if the user did not specify one?